CONTRIBUTING.md

```diff
@@ -35,24 +41,169 @@ You should receive a response within 24 hours. If for some reason you do not, pl
 
-## Policy
+### Policy
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
+
+## Develop for CBL-Mariner
+
+When starting to develop for CBL-Mariner, use the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo. This repository guides developers on using CBL-Mariner's tools to customize or add new packages or images. Once you have confirmed your change builds and functions as expected, consider whether it should be added to the core repo, [CBL-Mariner](https://github.com/microsoft/CBL-Mariner). Please see our [quickstart](toolkit/docs/quick_start/quickstart.md) for a tutorial and [building instructions](toolkit/docs/building/building.md) for an in-depth overview of building within CBL-Mariner. Please adhere to the [Pull Request guidelines](pull-request-guidelines) when contributing.
+
+### Packages
+
+CBL-Mariner packages live in either [SPECS](SPECS) or [SPECS-EXTENDED](SPECS-EXTENDED). CBL-Mariner packages that are in the `SPECS` directory have full support and coverage with timely CVE maintenance. Packages in `SPECS-EXTENDED` are for experimentation or proof-of-concept purposes only. `SPECS-EXTENDED` can be used as a staging area for iterating on packages with the possiblity of the package being graduated to `SPECS`.
+
+| Package Support Level | Published | Supported | Comments
+|:----------------------|:----------|:--------|:------------
+|SPECS-EXTENDED         |**Yes**    |No       | - Package needs a viable upstream source which actively addresses CVEs <br> - Package must not include project specific code
+|SPECS                  |**Yes**    |**Yes**  | - Package needs a viable upstream source which actively addresses CVEs <br> - Package must not include project specific code <br> - Package needs to offer value for multiple use cases
+
+When looking to graduate a package from `SPECS-EXTENDED` to `SPECS`, file a [GitHub issue](https://github.com/microsoft/CBL-Mariner/issues) highlighting the package's value and ensure that the following steps are completed for associated PRs.
+
+1. Increment the spec's `Release` value
+1. Add changelog entries "Package promoted from SPECS-EXTENDED to SPECS" and "License verified"
+1. Add a `%check` section to the spec if not already present and confirm it passes
+1. Ensure spec is free of references, macros, comments, etc. only applicable to other distros
+1. Complete the [PR checklist](#pr-checklist)
+1. Pass the GitHub checks
+
+### Toolkit
+
+We welcome tooling improvements. When contributing to the toolkit, please adhere to `golang` formatting as described by the [fmt](https://pkg.go.dev/fmt) package. To format using this package, you can run `make go-tidy-all` in your CBL-Mariner toolkit. For guidance building with the toolkit, see our [building instructions](toolkit/docs/building/building.md).
```

Comment:
```suggestion
We welcome tooling improvements. When contributing to the toolkit, please adhere to `golang` formatting as described by the [fmt](https://pkg.go.dev/fmt) package. To format using this package, you can run `make go-tidy-all` in your CBL-Mariner toolkit. For guidance on building with the toolkit, see our [building instructions](toolkit/docs/building/building.md).
```

Reply:
Thank you for your guidance **on** this . I will fix now ;).